### DISCUSSION TITLE
Guest account

### DISCUSSION THREAD

**Discussion by mittalyashu** [logchimp/logchimp#5] [Ideas]
Guest account allows the user to quickly submit a feature request _(a.k.a post)_ on LogChimp without creating an account but they will be required to submit the email address to avoid spam submissions.

Without having a proper account setup by the user, there are changes which can leads to spam that's why this feature will be toggle-able from the logchimp admin settings page.

_What do you think of it? Please, share your feedback._

**Comment by kaushikveluru** [comment]
I don't think there is any reason for non-registered users to participate in voting or submitting features on the site. 
They MUST be registered (need not be verified) in my opinion. 
For easier signup, we can look into OAuth like SignIn with Google/Twitter/Github etc.

**Comment by mittalyashu** [comment]
> I don't think there is any reason for non-registered users to participate in voting or submitting features on the site.
They MUST be registered (need not be verified) in my opinion.

Yes, I was also not sure to consider this feature or not. That's why I have added the 'discussion' label.

**Comment by mittalyashu** [comment]
> For easier signup, we can look into OAuth like SignIn with Google/Twitter/Github etc.

That's definitely the option planned to add in later version.
